spec/models/user_spec.rb

```diff
@@ -28,15 +31,15 @@
       end
 
       context 'when the email is not unique' do
-        before { create(:user, email:) }
+        before { create(:user, email:, password:, name:, birth:) }
 
         it 'is invalid' do
           expect(subject).to eq(false)
         end
       end
 
       context 'when the email is unique' do
-        before { create(:user, email: 'otheremail@example.com') }
+        before { create(:user, email: 'otheremail@example.com', password:, name:, birth:, username: Faker::Internet.username) }
```

Comment:
you don't need to pass `password`, `name`, `birth` nor `username` to this user. Those are already generated by the factory